### DISCUSSION TITLE
Trying to set up email

### DISCUSSION THREAD

**Discussion by hunterelele** [requarks/wiki#6525] [Help / Questions] [answered]
I'm trying to set up emails and keep getting this error when trying to send a test email:

Network error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

Everything is correctly done

**Comment by NGPixel** [comment, marked as answer]
You're using a reverse-proxy (e.g. Cloudlfare) which is blocking the requests.

**Comment by hunterelele** [comment]
I figured it out, out host was blocking port 25 and 445, once it was unblocked we can send emails
